release-tools/stage-release.sh

```diff
@@ -599,8 +601,9 @@ if [ -n "$PRE_LABEL" ]; then
     release_text="$SERIES$_BUILD_METADATA $PRE_LABEL $PRE_NUM"
     announce_template=openssl-announce-pre-release.tmpl
 else
+    release_type=$(std_release_type $VERSION)
     release_text="$release"
-    announce_template=openssl-announce-release.tmpl
+    announce_template=openssl-announce-release-$release_type.tmpl
```

Comment:
I haven't tracked down if possible to give wrong $VERSION. If yes, this would result in:
"announce_template=openssl-announce-release-.tmpl" since "std_release_type" function returns empty string if doesn't meet the regular expression criteria (to return "premium" or "public".